src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -138,6 +138,8 @@ public class CaseData implements MappableObject {
     private final Address specApplicantCorrespondenceAddressdetails;
     private final YesOrNo specRespondentCorrespondenceAddressRequired;
     private final Address specRespondentCorrespondenceAddressdetails;
+    private final YesOrNo specAoSRespondent2HomeAddressRequired;
```

Comment:
Since we already hit the too many parameters error in CaseData, may be worth considering a way to collapse these two fields into one, although it'll require front to use a complex field kind of 
specAoSRespondent2HomeAddress {
  required: YesOrNo,
  details: Address
}

Reply:
I think there will be refactor on the class later. Is it better to do this at that moment?

Reply:
We probably can leave it to another PR